docs/verbs/unhot.md

```diff
@@ -2,6 +2,11 @@
 
 Reverses a onehot encoding by collapsing the encoded columns into a single column and mapping the header values back to cell values. This operation reverses [onehot](./onehot.md).
 
+Additional options:
+
+- Keep source columns: leave the sources instead of replacing with the new collapsed column.
```

Comment:
should the name of these match their javascript keys? `keepSourceColumns`, `prefix`?